key-tester.js

```diff
@@ -115,7 +115,7 @@ function run(form_url, ga_code) {
         });
         $('#ssh-form').submit(function(e) {
             e.preventDefault();
-            var key = $('#form-ssh-key').val();
+            var key = rsa.sanitize($('#form-ssh-key').val());
```

Comment:
In fact sanitize is already used within `rsa.parse`. This is a mistake as I wanted to sanitize the key before sending it to formkeep but that's not the way. I'll fix this.